share/appdata/lutris.appdata.xml

```diff
@@ -1,35 +1,56 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component type="desktop">
```

Comment:
You are downgrading the format...

Re-run `appstream-util upgrade` on the file...